src/components/Buttons.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import NumberButton from './NumberButton';
+
+export default function Buttons({ onClick }) {
+  const numbers = [1, 2, 3, 4, 5];
+  return (
+    <div>
+      {numbers.map((i) => (
+        <NumberButton key={i} number={i} onClick={onClick}>{i}</NumberButton>
```

Comment:
```suggestion
        <NumberButton
          key={i}
          number={i}
          onClick={onClick}>
            {i}
        </NumberButton>
```

저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.

Reply:
좋은 꿀팁입니다 감사합니다!